PWGHF/Utils/utilsEvSelHf.h

```diff
@@ -194,4 +208,62 @@ struct HfEventSelection : o2::framework::ConfigurableGroup {
 };
 } // namespace o2::hf_evsel
 
+namespace o2::hf_evsel_mc
+{
+// MC collision rejection types
+enum McCollisionRejection {
+  None = 0,
+  Trigger,
+  TvxTrigger,
+  TimeFrameBorderCut,
+  ItsRoFrameBorderCut,
+  PositionZ,
+  NMcCollisionRejection
+};
```

Comment:
Looks like declaring a separate `enum` for MC is now redundant since it has become a subset of `EventRejection`.